CHANGELOG.md

```diff
@@ -7,13 +7,37 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
```

Comment:
Could you please update this changelog with only one [Unreleased] section? We just released one version so you may have to pull in and provide updated changelog.

Reply:
Yep, have done this now